cli/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "3.5.0"
 authors = ["Phylum, Inc. <engineering@phylum.io>"]
 edition = "2021"
 rust-version = "1.60"
+build = "build.rs"
```

Comment:
This is not required, right? The file should be picked up by default.

Reply:
Why? It serves no purpose as far as I can tell and I rarely see it added.

Reply:
To be explicit about the fact that a build script is, in fact, present and required; this way, if the file gets inadvertently deleted, Cargo will output an error:

```
> cargo check
   Compiling phylum-cli v3.5.0 (F:\Phylum\repos\cli\cli)
error: couldn't read cli\build.rs: The system cannot find the file specified. (os error 2)
```